source/dem/gear3_integrator.cc

```diff
@@ -81,15 +81,27 @@ for (auto particle = particle_handler.begin();
 template <int dim>
 void
 Gear3Integrator<dim>::integrate(
-  Particles::ParticleHandler<dim> & /* particle_handler */,
-  const Tensor<1, 3> & /* g */,
-  const double /* dt */,
-  std::vector<Tensor<1, 3>> & /* torque */,
-  std::vector<Tensor<1, 3>> & /* force */,
-  const std::vector<double> & /* MOI */,
+  Particles::ParticleHandler<dim> &particle_handler,
+  const Tensor<1, 3>              &g,
+  const double                     dt,
+  std::vector<Tensor<1, 3>>       &torque,
+  std::vector<Tensor<1, 3>>       &force,
+  const std::vector<double>       &MOI,
```

Comment:
Same question as for the explicit euler one. Anyway I will delete this integrator soon so it's not a big deal I think